frontend/src/app/Catalog/CatalogList/CatalogList.module.scss

```diff
@@ -0,0 +1,145 @@
+.root {
+	display: flex;
+	grid-area: list;
+	gap: 10px;
+	position: relative;
+	flex-direction: column;
+	border-right: 1.5px solid #eaeaea;
+	height: calc(100% + 48px);
```

Comment:
This height should be `100%` and not `calc(100% + 48px)`. A height greater than 100% causes the element to expand past the height of its parent before overflowing, which means the scrollbar gets cut off.